### DISCUSSION TITLE
Slack histogram horizontal axis

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD#3915] [General]
What is the horizontal axis?

I would have thought picoseconds, but the horizontal axis is not labeled...

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/2798822/9b5c6daa-25f6-42b7-adf4-e11c30bf3234)

This is from `make gui_place` in #3910

**Comment by oharboe** [comment]
Also, when I click on "Endpoint slack" upper left corner, it looks like there should have been more than one choice?

https://drive.google.com/file/d/1xCo_SBrhFjxZfQ-IOakjJyKaGkyZcZIP/view?usp=sharing

**Comment by maliberty** [comment]
It looks like 
![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/761514/202fee93-b91e-4c1c-a8da-65d0e27a213c)
 is in the wrong place  @AcKoucher any idea why?

**Reply by AcKoucher** [reply]
No idea. I'll look into it.
